src/foam/nanos/test/TestBorder.js

```diff
@@ -7,11 +7,11 @@
 foam.CLASS({
   package: 'foam.nanos.test',
   name: 'TestBorder',
-  extends: 'foam.u2.view.ScrollTableView',
+  extends: 'foam.u2.View',
 
   implements: ['foam.mlang.Expressions'],
 
-  requires: ['foam.nanos.test.Test'],
+  requires: ['foam.nanos.test.Test', 'foam.u2.view.ScrollTableView'],
```

Comment:
Please split one per line when more than one. 
```suggestion
  requires: [
    'foam.nanos.test.Test', 
    'foam.u2.view.ScrollTableView'
  ],
```

Reply:
Done.